mandatory/2-function-creation.js

```diff
@@ -16,7 +18,10 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
+function formatPercentage(num) {
+    
+      (num > 100)?   num.replace("100")   : `% ${num.toFixed(2)} `;
+}
```

Comment:
Really good to see the use of a ternary operator, that's quite advanced! The only things missing here is adding the % sign to numbers greater than 100, and a return statement, but apart from that this is good!

Reply:
why should I add a percent sign here ?

Reply:
Hey Zobeir, this was just so that it would return "100%" rather than "100". Purely a formatting thing!